### PR TITLE
[FIXED] document is not defined fix for server side rendering

### PR DESCRIPTION
**Pull request by egorvoronov**

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by egorvoronov** [comment]
CLA license agreed

**Comment by egorvoronov** [comment]
Seems that Travis tests are broken

File: src/Touch.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ const elementsFromPoint = ((typeof document !== 'undefined' && document.elements
     // return our results
     return elements;
 
-}).bind(document);
+}).bind(typeof document !== undefined ? document : null);
```
Comment (JoaoMosmann) [inline src/Touch.js]: `undefined` should be a string like `"undefined"`

**Pull request merged**